filelog.go

```diff
@@ -331,11 +342,7 @@ func (w *FileLogWriter) openLogFile() error {
 		return err
 	}
 
-	// Close any log file that may be open
```

Comment:
I'm unsure all the cases, but I'd be more comfortable leaving in a close here as well.